interpreter/php/php.go

```diff
@@ -342,6 +342,10 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 	vms.zend_string.val = 24
 	vms.zend_op.lineno = 24
 	switch {
+	case version >= phpVersion(8, 4, 0):
+		vms.zend_function.op_array_filename = 168
+		vms.zend_function.op_array_linestart = 176
+		vms.zend_function.Sizeof = 256
```

Comment:
It is enough for this to be largest offset +8. So 184 should be good. See the comment above regarding this.